docs/api.md

```diff
@@ -133,6 +133,20 @@ custom visualizations. After a `collectOnly` run, the `ticks` data is stored in
 const ticks = JSON.parse(fs.readFileSync('{pid}.0x/ticks.json', 'utf8'))
 ```
 
+`opts` is an object with optional properties:
+
+ - `inlined` - data about inlined functions. 0x stores this in the `meta.json`
+   file, which you can read like:
+
+   ```js
+   const { inlined } = JSON.parse(fs.readFileSync('{pid}.0x/meta.json', 'utf8'))
+   ```
+
+ - `mapFrames(frame)` - See [mapFrames](#mapframes-function).
+ - `pathToNodeBinary` - Path to the Node.js executable. Defaults to
```

Comment:
We might want to shorten the name if we expose this variable to e.g., `binaryPath`.

Reply:
I kinda agree but the main 0x function has an option by this same long name :(

Reply:
Oh, true. I forgot about that. Too bad. We have to wait for a major to rename it.